### ISSUE TITLE
setting done as False in robomimic

### ISSUE THREAD

**Issue by lakshitadodeja** [irom-princeton/dppo#32]
Hi, 

I see that in robomimic wrapper. the done value is being automatically set as False. If we want to change it to its true value, is returning the actual done value enough ? Or are there some other changes in the training code that we need to handle for trajectories that terminate early ?

**Comment by allenzren** [comment]
Hi! I am not sure exactly right now. Could you give a try and see if the done value from robosuite corresponds to termination?

**Comment by lakshitadodeja** [comment]
I tried to check if the task for successful from robomimic and passed done as - 

```
truncated = self.env.env._check_success()
done = terminated or truncated
```

The success rate from the evaluation increased with that (the one with true done is the one with changes) - 
![dppo](https://github.com/user-attachments/assets/746822bf-e3cd-46cf-8cf3-7fe1d92b2ec2)


Though the number of eval episodes also increased, probably because some episodes ended early ?
![W B Chart 1_8_2025, 8_10_31 PM](https://github.com/user-attachments/assets/70cbab4b-7018-4a4f-a545-4c5aa171a0c3)

Is this expected ?

**Comment by allenzren** [comment]
Sorry about the late response. The increase in the number of eval episodes makes sense to me since the episode can terminate early now.

But I don't understand the reason that the eval success rate also increases. The episode is deemed successful if the task is completed at any timestep (reward=1). My suggestion would be to inspect carefully the numbers around where the success rate is calculated https://github.com/irom-princeton/dppo/blob/e7f73dffc131570ef7129d5ed1bc98a05cf030ab/agent/finetune/train_ppo_diffusion_agent.py#L184
